frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -89,6 +88,14 @@ export const DetailsCard_ = connect(mapStateToProps)(({
         <DetailsBody>
           {openshiftFlag ? (
             <>
+              <DetailItem
+                key="apiurl"
+                title="Cluster API address"
+                isLoading={!infrastructureLoaded && !infrastructureError}
+                error={!infrastructurePlatform}
```

Comment:
shouldnt it be `error={!!infrastructureError || !getInfrastructureAPIURL(infrastructureData)}` ?

Reply:
done